File: dot-net/src/DIP.Controller/DIP.Controller/DinnersController.cs
using DIP.Controller.Model;

namespace DIP.Controller
{
    public class DinnersController
    {
        private readonly DinnerRepository _dinnerRepository;

        public DinnersController()
        {
            _dinnerRepository = new DinnerRepository();
        }

        public Dinner GetById(int id)
        {
            return _dinnerRepository.GetById(id);
        }

    }
}

File: java/dip-controller/src/DinnersController.java

public class DinnersController {

    private final DinnerRepository dinnerRepository;

    public DinnersController()
    {
        this.dinnerRepository = new DinnerRepository();
    }

    public Dinner getById(int id)
    {
        return this.dinnerRepository.getById(id);
    }
}